### ISSUE TITLE
Measurement still aligns best to "Front Edge" limiting, but should not (Start vs. Present)!?

### ISSUE THREAD

**Issue by Weissrolf** [CXWorld/CapFrameX#296]
> The majority of metrics are changed to use the time that the CPU started working on a frame as the reference point instead of the present() call, with values that are more aligned to measuring the quality of graphics applications (e.g., latency and duration of interaction and displayed frames).

According to the latest change-notes, which include Unwinder's (RTSS) explanation, the new version of CapFrameX should align better with latency based fps limiting methods (Back Edge, Async), but it still aligns better with RTSS' "Front Edge" (framepacing) limiting.

This seems to suggest that CFX is still using Present-to-Present based measurement instead of Start-to-Start!?

**Comment by DevTechProfile** [comment]
Yes, CapFrameX now using Start-to-Start. You must force PresentMon to fall back to the old approach actually by using "--v1_metrics" command line option. We don't do that. 

The question is, how did you evaluate that "it still aligns better with RTSS' "Front Edge" (framepacing) limiting"?

**Comment by Weissrolf** [comment]
Present-to-Present always aligned straight with Front Edge limiting, which means that old CapFrameX versions fit Front Edge better. With the new version and change to Start-to-Start measurement I expected Front Edge to now be at a disadvantage and other limiting methods (like Back Edge) to report more straight numbers. But that doesn't happen with CapFrameX, whereas with RTSS it does.

**Comment by DevTechProfile** [comment]
This is definitely an interesting topic. Nevertheless, I would recommend to open an issue on the PresentMon GitHub repo directly. The PM team can answer these questions better than me.
